bigtable/beam/change-streams/src/main/java/ChangeStreamsHelloWorld.java

```diff
@@ -0,0 +1,151 @@
+import com.google.cloud.bigtable.data.v2.models.DeleteCells;
+import com.google.cloud.bigtable.data.v2.models.DeleteFamily;
+import com.google.cloud.bigtable.data.v2.models.Entry;
+import com.google.cloud.bigtable.data.v2.models.SetCell;
+import com.google.protobuf.ByteString;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.beam.runners.dataflow.options.DataflowPipelineOptions;
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.gcp.bigtable.BigtableIO;
+import org.apache.beam.sdk.options.Default;
+import org.apache.beam.sdk.options.Description;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.transforms.DoFn;
+import org.apache.beam.sdk.transforms.FlatMapElements;
+import org.apache.beam.sdk.transforms.ParDo;
+import org.apache.beam.sdk.values.KV;
+import org.apache.beam.sdk.values.TypeDescriptors;
+import org.joda.time.Instant;
+
+public class ChangeStreamsHelloWorld {
+
+  public static void main(String[] args) {
+    BigtableOptions options =
+        PipelineOptionsFactory.fromArgs(args).withValidation().as(BigtableOptions.class);
+    Pipeline p = Pipeline.create(options);
+
+    final Instant startTime = Instant.now();
+
+    p.apply(
+            "Read Change Stream",
+            BigtableIO.readChangeStream()
+                .withProjectId(options.getBigtableProjectId())
+                .withInstanceId(options.getBigtableInstanceId())
+                .withTableId(options.getBigtableTableId())
+                .withAppProfileId(options.getBigtableAppProfile())
+                .withStartTime(startTime))
+        .apply(
+            "Flatten Mutation Entries",
+            FlatMapElements.into(TypeDescriptors.strings())
+                .via(ChangeStreamsHelloWorld::mutationEntriesToString))
+        .apply(
+            "Print mutations",
+            ParDo.of(
+                new DoFn<String, Void>() { // a DoFn as an anonymous inner class instance
+                  @ProcessElement
+                  public void processElement(@Element String mutation) {
+                    System.out.println("Change captured: " + mutation);
+                  }
+                }));
+    p.run();
+  }
+
+  static List<String> mutationEntriesToString(KV<ByteString, ChangeStreamMutation> mutationPair) {
+    List<String> mutations = new ArrayList<>();
+    String rowKey = mutationPair.getKey().toStringUtf8();
+    ChangeStreamMutation mutation = mutationPair.getValue();
+    MutationType mutationType = mutation.getType();
+    for (Entry entry : mutation.getEntries()) {
+      if (entry instanceof SetCell) {
+        mutations.add(setCellToString(rowKey, mutationType, (SetCell) entry));
+      } else if (entry instanceof DeleteCells) {
+        mutations.add(deleteCellsToString(rowKey, mutationType, (DeleteCells) entry));
+      } else if (entry instanceof DeleteFamily) {
```

Comment:
what if it's none of these, or is an error?

Reply:
added else